plugin/assets/src/customizer/components/color-control/style.css

```diff
@@ -0,0 +1,27 @@
+.material-design-color__picker {
+	align-items: center;
+	display: flex;
+	margin-bottom: 15px;
+
+	& .components-base-control .components-base-control__field {
```

Comment:
Is there a fallback for browsers that dont support nested css?

Reply:
@rodydavis This is turned into `.material-design-color__picker .components-base-control .components-base-control__field` in the compiled file